Resources/Prototypes/Reagents/cleaning.yml

```diff
@@ -67,6 +67,19 @@
   viscosity: 0.5
   tileReactions:
     - !type:SpillTileReaction
+  reactiveEffects:
+    Acidic:
```

Comment:
Why should touching this reagent (or slipping on a puddle of it and falling on it and having it transfer) mute you?

Reply:
removed